app/view_models/sign_up_completions_show.rb

```diff
@@ -2,11 +2,21 @@ class SignUpCompletionsShow
   include ActionView::Helpers::AssetTagHelper
   include ActionView::Helpers::TagHelper
 
-  def initialize(loa3_requested:)
+  def initialize(loa3_requested:, decorated_session:)
     @loa3_requested = loa3_requested
+    @decorated_session = decorated_session
   end
 
-  attr_reader :loa3_requested
+  attr_reader :loa3_requested, :decorated_session
+
+  SORTED_ATTRIBUTE_MAPPING = [
+    [%i[given_name family_name], :full_name],
+    [%i[address1 address2 city state zipcode], :address],
```

Comment:
per our slack discussion, it looks like `SamlRequestPresenter` is mapping all the address attributes directly to `address` so this can just be `[:address], :address` (unsure why the feature tests didn't catch that?)